geaflow/geaflow-dsl/geaflow-dsl-plan/src/main/java/com/antgroup/geaflow/dsl/udf/table/string/ConcatWS.java

```diff
@@ -14,14 +14,20 @@
 
 package com.antgroup.geaflow.dsl.udf.table.string;
 
+import com.antgroup.geaflow.common.binary.BinaryString;
 import com.antgroup.geaflow.dsl.common.function.Description;
 import com.antgroup.geaflow.dsl.common.function.UDF;
+import java.util.Objects;
 import org.apache.commons.lang3.StringUtils;
 
 @Description(name = "concat_ws", description = "Concat strings to one by the separator string.")
 public class ConcatWS extends UDF {
 
     public String eval(String separator, String... args) {
-        return StringUtils.join(separator, args);
+        return StringUtils.join(args, separator);
+    }
+
+    public String eval(String separator, BinaryString... args) {
```

Comment:
Use BinaryString as the return type

Reply:
Hi @liuxiaocs7 can you take some time to resolve the rest comment?

Reply:
Hi, @pengzhiwei2018, i have resolved comments, could you take a look when you are free.